lib/KCard/__test__/KCard.spec.js

```diff
@@ -0,0 +1,110 @@
+    const wrapper = makeWrapper({
+      propsData: { headingLevel: 4, title: 'sample title prop', to: { path: '/some-link' } },
+    });
+
+    const heading = wrapper.find('h4');
+    expect(heading.exists()).toBe(true);
+  });
+
+  it('renders the correct accessibility  structure when title slot is passed', () => {
+    const wrapper = makeWrapper({
+      propsData: { to: { path: '/some-link' }, headingLevel: 4 },
+      slots: {
+        title: 'Test Title',
+      },
+    });
+    const titleSlots = wrapper.find('h4');
+    expect(titleSlots.exists()).toBe(true);
+    const firstElement = wrapper.find('li');
+    expect(firstElement.element.tagName.toLowerCase()).toBe('li');
+    expect(firstElement.exists()).toBe(true);
+    const secondElement = wrapper.find('li > :first-child');
+    expect(secondElement.exists()).toBe(true);
+    expect(secondElement.element.tagName.toLowerCase()).toBe('h4');
+    const thirdElement = wrapper.find('h4 > :first-child');
+    expect(thirdElement.element.tagName.toLowerCase()).toBe('a');
+    expect(thirdElement.exists()).toBe(true);
+  });
+
+  it('renders the correct accessibility structure when title pass as props', () => {
+    const wrapper = makeWrapper({
+      propsData: { to: { path: '/some-link' }, headingLevel: 4, title: 'Test Title' },
+      slots: {},
+    });
+    const firstElement = wrapper.find(':first-child');
+    expect(firstElement.exists()).toBe(true);
+    expect(firstElement.element.tagName.toLowerCase()).toBe('li');
+
+    const secondElement = wrapper.find('li > :first-child');
+    expect(secondElement.exists()).toBe(true);
+    expect(secondElement.element.tagName.toLowerCase()).toBe('h4');
+
+    const thirdElement = wrapper.find('h4 > :first-child');
+    expect(thirdElement.exists()).toBe(true);
+    expect(thirdElement.element.tagName.toLowerCase()).toBe('a');
+  });
+
+  it('should not navigate on long click', async () => {
+    const wrapper = makeWrapper({
+      propsData: { to: { path: '/some-link' }, title: 'sample title prop' },
+    });
+
+    await wrapper.find('li').trigger('mousedown');
+    await new Promise(resolve => setTimeout(resolve, 500));
+    await wrapper.find('li').trigger('mouseup');
+    expect(wrapper.vm.$router.currentRoute.path).not.toBe('/some-link');
+  });
+
+  it('should navigate on quick click', async () => {
+    const wrapper = makeWrapper({
+      propsData: { to: { path: '/some-link' }, title: 'sample title ' },
+    });
+
+    await wrapper.find('li').trigger('mousedown');
+    await new Promise(resolve => setTimeout(resolve, 100));
+    await wrapper.find('li').trigger('mouseup');
+
+    expect(wrapper.vm.$router.currentRoute.path).toBe('/some-link');
+  });
+
+  describe('it renders slotted content', () => {
+    it('renders slotted content with belowTitle slot', () => {
+      const wrapper = makeWrapper({
+        propsData: { to: { path: '/some-link' }, title: 'sample title ' },
+        slots: {},
```

Comment:
I think we missed passing slots here. Here we have the intention of test slotted content, but we are not passing anything to the slots prop.